lib/kino_db/connection_cell.ex

```diff
@@ -362,27 +365,53 @@ defmodule KinoDB.ConnectionCell do
       database: attrs["database"]
     ]
 
-    opts =
-      if attrs["use_ssl"] do
-        opts ++ [ssl: attrs["use_ssl"]]
-      else
-        opts
-      end
-
     if attrs["use_ipv6"] do
       opts ++ [socket_options: [:inet6]]
     else
       opts
     end
   end
 
+  defp postgres_and_mysql_options(attrs) do
+    if attrs["use_ssl"] do
+      cacertfile = attrs["cacertfile"]
+
+      ssl_opts =
+        if cacertfile && cacertfile != "" do
+          [cacertfile: cacertfile]
+        else
+          [cacerts: quote(do: :public_key.cacerts_get())]
+        end
+
+      [ssl: ssl_opts]
+    else
+      []
+    end
+  end
+
   defp sqlserver_options(attrs) do
+    opts =
+      if attrs["use_ssl"] do
+        cacertfile = attrs["cacertfile"]
+
+        ssl_opts =
+          if cacertfile && cacertfile != "" do
+            [cacertfile: cacertfile]
+          else
+            [cacerts: quote(do: :public_key.cacerts_get())]
+          end
+
+        [ssl: true, ssl_opts: ssl_opts]
```

Comment:
Gah, we should update TDS. @wojtekmach? :P

Reply:
yup, I'm happy to work on that!